protecode/model.py

```diff
@@ -50,7 +50,10 @@ def custom_data(self):
         return self.raw.get('custom_data')
 
     def __repr__(self):
-        return f'{self.__class__.__name__}: {self.name()}({self.product_id()})'
+        return f'{self.__class__.__name__}: {self.display_name()}({self.product_id()})'
+
+    def __str__(self):
```

Comment:
according to https://docs.python.org/3/reference/datamodel.html#object.__repr__, this should not be necessary. instead, consider changing the super-class's `__str__` to `__repr__`